rules/aip0131/message_rules_test.go

```diff
@@ -19,7 +19,7 @@ import (
 
 	"github.com/jhump/protoreflect/desc"
 	"github.com/jhump/protoreflect/desc/builder"
-	"google.golang.org/genproto/protobuf/field_mask"
+	fpb "google.golang.org/genproto/protobuf/field_mask"
```

Comment:
Why were these lint errors exactly?

Reply:
Protocol buffer packages should have short names that end in "pb" to distinguish with other canonical Go packages.